SU-SWS/stanford_profile_helper
# D8CORE-3263 Gallery paragraph styling

# READY FOR REVIEW

# Summary
- Styles for image gallery paragraph type
- Dynamic scss compiling
   - Now simply adding a new file will generated a compiled css file. no need to edit the webpack file. it will treat any file that begins with an underscore as a partial and not use that as an entry point.
- Dynamic library generation
   - Each css file will create a respective library. No more need to edit the libraries.yml file each time.

# Need Review By (Date)
- 1/14

# Urgency
- medium

# Steps to Test
1. checkout this branch
2. clear caches
3. verify nothing looks different.

Other steps will be separate in https://github.com/SU-SWS/stanford_profile/pull/347

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
